webstudio-is/webstudio
# refactor: move expressions to props in embed template

Ref https://github.com/webstudio-is/webstudio/issues/2532

We decided to treat expressions as props bindings
so it makes sense to move expression data to props as well. This will make expression and action simetrical and simplify data sources management.

As you can see all expressions have same "expression" name now. In another PR will change to prop name.

## Testing

1. check newly created Form states
2. Check all radix components states

## Code Review

  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review


## Before merging
